src/lib/motion-start/motion/features/layout/MeasureLayoutWithContext.svelte

```
@@ -73,11 +85,22 @@ let hasCompletedInitialPrepass = false;
-	[() => props.layoutDependency, () => props.drag, () => props.visualElement?.projection, () => props.isPresent],
-	([], [prevLayoutDependency, , , prevIsPresent]) => {
-		const { layoutDependency, visualElement, isPresent } = props;
+	[
+		() => renderSnapshot,
+		() => props.layoutDependency,
+		() => props.ambientLayoutVersion,
+		() => props.drag,
+		() => props.visualElement?.projection,
+		() => props.isPresent,
+	],
+	([], [, prevLayoutDependency, prevAmbientLayoutVersion, , , prevIsPresent]) => {
```

Biome (2.5.3)

[error] Unexpected empty array pattern.

(lint/correctness/noEmptyPattern)